python/src/common.cpp

```diff
@@ -57,7 +57,7 @@ void common(py::module& m)
         "Return `True` if DOLFIN is configured with mpi4py");
   m.def("has_parmetis", &dolfin::has_parmetis);
   m.def("has_scotch", &dolfin::has_scotch);
-  m.def("has_petsc_complex", &dolfin::has_petsc_complex,
+  m.def("complex_mode", &dolfin::has_petsc_complex,
```

Comment:
Why is `has_petsc_complex` wrapped as `complex_mode`? This is just confusing. It should be wrapped as `has_petsc_complex`.